### PR TITLE
Store artifacts in cloud

### PR DESCRIPTION
**Pull request by Kalaiselvi84**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, press enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind breaking
> /kind bug
> /kind cleanup
> /kind documentation
> /kind feature
> /kind hotfix

/kind cleanup
/kind feature

**What this PR does / Why we need it**:

**Which issue(s) this PR fixes**:

work on #2460 

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/governance/templates/release_issue.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ and copy it into a release issue. Fill in relevant values, found inside {}
 - [ ] `git checkout main && git pull --rebase upstream main`
 - [ ] Run `make release-deploy-site`
       - For example, if you are creating the {version} release, then this would deploy the {version}-1 service (release minus one, and then replace dots with dashes)
-- [ ] Run `make build-release` to ensure all example images exist on us-docker.pkg.dev/agones-images/examples.
+- [ ] Run `make build-release` to ensure all example images exist on us-docker.pkg.dev/agones-images/examples and to store the artifacts in cloud storage.
```
Comment (markmandel) [inline docs/governance/templates/release_issue.md:36]: This looks great! The only other thought I had to add, is to mention when to download the files form cloud storage so they can be attached to the Github release. 👍🏻

Comment (Kalaiselvi84) [reply docs/governance/templates/release_issue.md]: Updated artifacts storage related steps in release checklist

**Comment by markmandel** [comment]
This looks good! I was going to suggest let's merge this after the release next Thursday - as then we can add more steps to it before the next release (primarily also pushing the images here that also get built).

Sound good?

**Comment by Kalaiselvi84** [comment]
> This looks good! I was going to suggest let's merge this after the release next Thursday - as then we can add more steps to it before the next release (primarily also pushing the images here that also get built).
> 
> Sound good?

sounds good!

**Comment by markmandel** [comment]
Can you please go back and `git rebase main` on the latest `upstream/main` on this PR and push this back up. It's very messy with all the extra commits.

**Pull request closed without merging**